vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs

```diff
@@ -142,11 +139,8 @@ type internal FSharpCompletionProvider
             sortedDeclItems |> Array.iteri (fun number declItem ->
                 let glyph = CommonRoslynHelpers.FSharpGlyphToRoslynGlyph (declItem.Glyph, declItem.Accessibility)
                 let name =
-                    match entityKind, declItem.NamespaceToOpen with
-                    | Some EntityKind.Attribute, _ when declItem.IsAttribute && declItem.Name.EndsWith "Attribute"  ->
-                        declItem.Name.[0..declItem.Name.Length - attributeSuffixLength - 1]
-                    | _, Some namespaceToOpen ->
-                        sprintf "%s (open %s)" declItem.Name namespaceToOpen
+                    match declItem.NamespaceToOpen with
+                    | Some namespaceToOpen -> sprintf "%s (open %s)" declItem.Name namespaceToOpen
```

Comment:
@KevinRansom it appears in the completion list for items which are in namespaces/modules which aren't open.

I think it is fine to leave it untranslated because it shows the actual F# syntax. Any other opinion?